docs/modules/plugins/pages/plugin-mobitru.adoc

```diff
@@ -0,0 +1,62 @@
+:test-cloud-name: Mobitru
+:test-cloud-key: mobitru
+:test-cloud-link: https://mobitru.com/
+:mobitru-guide-link: https://app.mobitru.com/static/mstfdocs/public/Mobitru_User_Guide.pdf
+
+include::partial$test-cloud-header.adoc[]
+
+== Profiles
+
+The plugin defines profile with a set of the preconfigured properties. In order to simplify work with Mobitru the plugin does following steps:
+.Waits for the requested device to be available using https://app.mobitru.com/wiki/api/#api-Device-TakeDeviceDesiredCapabilities[API call] and takes it in the usage.
```

Comment:
```suggestion
.Waits for the requested device to be available using https://app.mobitru.com/wiki/api/#api-Device-TakeDeviceDesiredCapabilities[REST API call] and takes it in the usage.
```

Reply:
maybe just HTTP call? otherwise are you sure that mobitru API follows REST principles?